xml/containers_basics.xml

```diff
@@ -1,78 +1,202 @@
-  </listitem>
-  <listitem>
-   <para>
-    Containers are not secure, and the overall security depends on the host
-    system. Containerized applications can be secured through &aa; or &selnx;
-    profile.
-   </para>
-  </listitem>
- </itemizedlist>
+  <title>Introduction to Linux Containers</title>
+  <info>
+    <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+      <dm:bugtracker/>
+    </dm:docmanager>
+  </info>
+  <para>
+    Linux containers offer a lightweight virtualization method to run multiple
+    virtual environments (containers) simultaneously on a single host. Unlike
+    technologies like &xen; or &kvm;, where the processor simulates a complete
+    hardware environment and controls virtual machines, containers provide
+    virtualization at the operating system level, where the kernel controls the
+    isolated containers.
+  </para>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Advantages of Using Containers</title>
+    <listitem>
+      <para>
+        Containers make it possible to solate applications through self-contained units.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers provide nearly native performance, as containers manage
+        allocation of resources in real time.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        It is possible to control network interfaces and apply resources inside
+        containers through <xref linkend="gloss-vt-lxc-cgroups"/>.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <itemizedlist mark="bullet" spacing="normal">
+    <title>Limitations of Containers</title>
+    <listitem>
+      <para>
+        Containers run inside the host system's kernel, and they cannot use
+        different kernels or different kernel versions.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Only Linux-based applications can be containerized.
+      </para>
+    </listitem>
+    <listitem>
+      <para>
+        Containers are not secure, and the overall security depends on the host
+        system. Containerized applications can be secured through &aa; or &selnx;
+        profile.
+      </para>
+    </listitem>
+  </itemizedlist>
+  <sect1 xml:id="sec-concepts-workflow">
+    <title>Key Concepts and Brief Introduction to &docker;</title>
+    <para>
+    &deng; is a well-established and mature container technology. While <xref
+    linkend="cha-docker-overview" /> provides an in-depth information on &deng;,
+    this chapter offers a quick introduction to key concepts and a basic
+    procedure of creating a container image an using it to run a container.
+    </para>
+    <para>
+      The basic &docker; workflow is as follows:
+    </para>
+    <para>
+      Registry -> Base image -> Dockerfile -> Custom image -> Container
+      <!--TODO replace with a figure-->
+    </para>
+
+    <para>
+      Running a container on a local machine or cloud service, usually involves the following steps:
```

Comment:
This may be true for running it locally, but for remote machines or dedicated workers it's more likely like this:
On the local host:
- Write a `Dockerfile` to build an image
- Test the image
- Push the image to a registry

On the worker:
- Pull the image
- Start containers using the image

Reply:
Good point! But it's better to keep the intro short and simple and cover more advanced topics elsewhere. I added this to my to-do list.